Tranga/MangaConnectors/MangaKatana.cs

```diff
@@ -154,7 +154,7 @@ public override Chapter[] GetChapters(Manga manga, string language="en")
 		//Return Chapters ordered by Chapter-Number
 		List<Chapter> chapters = ParseChaptersFromHtml(manga, requestUrl);
 		Log($"Got {chapters.Count} chapters. {manga}");
-		return chapters.OrderBy(chapter => Convert.ToSingle(chapter.chapterNumber, numberFormatDecimalPoint)).ToArray();
+		return chapters.OrderBy(chapter => chapter.chapterNumber ?? chapter.volumeNumber).ToArray();
```

Comment:
We don't order by volume number. Chapters are always numbered 1-x regardless of volume (so there isn't a v1c1 and v2c1)

Reply:
I didn't seem to get the idea. Reverted this.